collection-scripts/gather_edpm_sos

```diff
@@ -50,7 +50,7 @@ if [[ -n "$SOS_EDPM_PLUGINS" ]]; then
 fi
 
 SSH () {
-    ssh -i "$key_path" "${username}@${address}" -o StrictHostKeyChecking=accept-new "$@"
+    ssh -n -i "$key_path" "${username}@${address}" -o StrictHostKeyChecking=accept-new "$@"
```

Comment:
ack based on https://linux.die.net/man/1/ssh
```
-n' Redirects stdin from /dev/null (actually, prevents reading from stdin). This must be used when ssh is run in the background. A common trick is to use this to run X11 programs on a remote machine. For example, ssh -n shadows.cs.hut.fi emacs & will start an emacs on shadows.cs.hut.fi, and the X11 connection will be automatically forwarded over an encrypted channel. The ssh program will be put in the background. (This does not work if ssh needs to ask for a password or passphrase; see also the -f option.) 
```

i agree this looks corrccet

i initially thought this was 
```
-N' Do not execute a remote command. This is useful for just forwarding ports (protocol version 2 only).
```

which would obviously not work.

nice spot.

Reply:
Great find @gibizer !!!